### PR TITLE
Added default metric port in flytescheduler

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR
Currently flytescheduler use 10253 as default metric port, But all other component use 10254 as default,

Replace port 10253 -> 10254 in flytescheduler for metric

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [x] Unit tests added
 - [x] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/1864

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Comment by pmahindrakar-oss** [comment]
Can we also test out if metrics are getting published correctly and prometheus is able to scrape and get the data.

**Review by EngHabu — APPROVED** [review]

**Comment by evalsocket** [comment]
@pmahindrakar-oss port is working fine in both services
